.gitignore

```diff
@@ -10,6 +10,9 @@ dist/
 # Docs generation
 docs/_build/
 
+# local folder
+src/pypromice/local
```

Comment:
I don't have this directory in my local copy, but I could use something similar to this for various testing scripts. So I'd be happy for this to be kept as part of this PR.

Reply:
This folder was supposed to host the local copy of the flag and adjustment files and was created, if not existing, the first time flagNAN function is used. But now you recommended to not keep a local copy of these files and to read_csv from url. 
So it is not needed anymore.

Reply:
Okay, I'll remove it and add it to the next commit.